libs/SalesforceSDK/src/com/salesforce/androidsdk/auth/idp/IDPManager.kt

```diff
@@ -26,34 +26,41 @@
  */
 package com.salesforce.androidsdk.auth.idp
 
-import android.app.Activity
 import android.content.Context
 import android.content.Intent
-import android.webkit.WebView
 import com.salesforce.androidsdk.accounts.UserAccount
 import com.salesforce.androidsdk.app.SalesforceSDKManager
-import com.salesforce.androidsdk.auth.idp.IDPSPMessage.*
+import com.salesforce.androidsdk.auth.idp.IDPSPMessage.Companion.ACTION_KEY
+import com.salesforce.androidsdk.auth.idp.IDPSPMessage.IDPToSPRequest
+import com.salesforce.androidsdk.auth.idp.IDPSPMessage.IDPToSPResponse
+import com.salesforce.androidsdk.auth.idp.IDPSPMessage.SPToIDPRequest
+import com.salesforce.androidsdk.auth.idp.IDPSPMessage.SPToIDPResponse
+import com.salesforce.androidsdk.auth.idp.interfaces.IDPAuthCodeActivity
 import com.salesforce.androidsdk.auth.idp.interfaces.IDPManager.Status
 import com.salesforce.androidsdk.auth.idp.interfaces.IDPManager.StatusUpdateCallback
 import com.salesforce.androidsdk.util.LogUtil
 import com.salesforce.androidsdk.util.SalesforceSDKLogger
 
 
 /**
- * Class to capture state related to IDP initiated login flow
+ * Class to capture state related to IDP-SP login in the IDP app
  * e.g. the context (activity) that started it
+ *      the web view to get the auth code in
  */
-internal class IDPInitiatedLoginFlow private constructor(context:Context, val user:UserAccount, val spConfig: SPConfig, val onStatusUpdate:(Status) -> Unit) : ActiveFlow(context) {
+internal class IDPLoginFlow(context:Context, val user:UserAccount, val spConfig: SPConfig, val onStatusUpdate:(Status) -> Unit) : ActiveFlow(context) {
+
+    var authCodeActivity: IDPAuthCodeActivity? = null
+
     companion object {
-        val TAG = IDPInitiatedLoginFlow::class.java.simpleName
+        val TAG = IDPLoginFlow::class.java.simpleName
```

Comment:
Renamed the flow class, since we create a flow class during SP initiated login also.